inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
+    sources: List[str] = field(default_factory=list)
     gpg_key_uri: Optional[str] = field(default=None)
     gpg_key_name: Optional[str] = field(default=None)
```

Comment:
**Suggestion:** Ensure that the source_list_file_name field is initialized consistently across all related data classes. [consistency]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
class ApplicationAddSourceParameters:
    source_list_file_name: str = field(default="")
    sources: List[str] = field(default_factory=list)
    gpg_key_uri: Optional[str] = field(default=None)
    gpg_key_name: Optional[str] = field(default=None)
```